swap/src/cli/command.rs

```diff
@@ -208,3 +437,484 @@ fn parse_monero_address(s: &str) -> Result<monero::Address> {
+                    monero_daemon_address: DEFAULT_STAGENET_MONERO_DAEMON_ADDRESS.to_string(),
+                    tor_socks5_port: DEFAULT_SOCKS5_PORT,
+                },
+            }
+        }
+
+        pub fn cancel_testnet_defaults() -> Self {
+            Self {
+                env_config: env::Testnet::get_config(),
+                debug: false,
+                data_dir: data_dir_path_cli().join(TESTNET),
+                cmd: Command::Cancel {
+                    swap_id: Uuid::from_str(SWAP_ID).unwrap(),
+                    force: false,
+                    bitcoin_electrum_rpc_url: Url::from_str(DEFAULT_ELECTRUM_RPC_URL_TESTNET)
+                        .unwrap(),
+                    bitcoin_target_block: DEFAULT_BITCOIN_CONFIRMATION_TARGET_TESTNET,
+                },
+            }
+        }
+
+        pub fn cancel_mainnet_defaults() -> Self {
+            Self {
+                env_config: env::Mainnet::get_config(),
+                debug: false,
+                data_dir: data_dir_path_cli().join(MAINNET),
+                cmd: Command::Cancel {
+                    swap_id: Uuid::from_str(SWAP_ID).unwrap(),
+                    force: false,
+                    bitcoin_electrum_rpc_url: Url::from_str(DEFAULT_ELECTRUM_RPC_URL).unwrap(),
+                    bitcoin_target_block: DEFAULT_BITCOIN_CONFIRMATION_TARGET,
+                },
+            }
+        }
+
+        pub fn refund_testnet_defaults() -> Self {
+            Self {
+                env_config: env::Testnet::get_config(),
+                debug: false,
+                data_dir: data_dir_path_cli().join(TESTNET),
+                cmd: Command::Refund {
+                    swap_id: Uuid::from_str(SWAP_ID).unwrap(),
+                    force: false,
+                    bitcoin_electrum_rpc_url: Url::from_str(DEFAULT_ELECTRUM_RPC_URL_TESTNET)
+                        .unwrap(),
+                    bitcoin_target_block: DEFAULT_BITCOIN_CONFIRMATION_TARGET_TESTNET,
+                },
+            }
+        }
+
+        pub fn refund_mainnet_defaults() -> Self {
+            Self {
+                env_config: env::Mainnet::get_config(),
+                debug: false,
+                data_dir: data_dir_path_cli().join(MAINNET),
+                cmd: Command::Refund {
+                    swap_id: Uuid::from_str(SWAP_ID).unwrap(),
+                    force: false,
+                    bitcoin_electrum_rpc_url: Url::from_str(DEFAULT_ELECTRUM_RPC_URL).unwrap(),
+                    bitcoin_target_block: DEFAULT_BITCOIN_CONFIRMATION_TARGET,
+                },
+            }
+        }
+
+        pub fn with_data_dir(mut self, data_dir: PathBuf) -> Self {
+            self.data_dir = data_dir;
+            self
+        }
+
+        pub fn with_debug(mut self) -> Self {
+            self.debug = true;
+            self
+        }
+    }
+
+    fn data_dir_path_cli() -> PathBuf {
+        system_data_dir().unwrap().join("cli")
+    }
```

Comment:
It would be nice to have all of these helpers below the tests. When encountering the `tests` module, I am primarily interested in the tests, not the helpers that make the tests easier to read.